src/views/MoviesPage.vue

```diff
@@ -45,17 +49,22 @@ const selectOptions = computed(() => {
       >Category</BaseSelect
     >
   </div>
-  <template v-if="!moviesStore.loading">
-    <div class="movies-list">
+  <template v-if="filteredTitleMovives.length > 0">
+    <div v-if="!uiStore.moviesLoading" class="movies-list">
       <MovieCard
         v-for="movie in filteredTitleMovives"
         :key="movie.id"
         :movie="movie"
         :to="{ name: 'SingleMoviePage', params: { id: movie.id } }"
       />
     </div>
+    <div v-else><LoadingSpinner /></div>
   </template>
-  <template v-else> Loading... </template>
+  <template v-else
```

Comment:
we don't need `<template>` here
```suggestion
  <NoResults v-else
      >Sorry we have no movies to show you at the moment</NoResults
```

Reply:
I thought it just looked more consistent but you are right functionally it is not bringing anything and redundant